docs/_posts/2018-07-30-conversion-to-go-modules.md

```diff
@@ -31,7 +31,7 @@ Since there has been no one official solution that addresses all of these issues
 
 ## `vgo` and the new Go modules
 
-Despite the growing acceptance of `dep` as _the_ dependency management tool for Golang, in mid-2018 a [new proposal](https://blog.golang.org/versioning-proposal) emerged for managing Golang project dependencies, known as `vgo` or Versioned Go Modules. The proposal was accepted, and as of [`Go1.11`](https://tip.golang.org/doc/go1.11#modules) modules will be available as an alternative to `GOPATH`, with integrated support for versioning and package distribution. Go modules are currently available as part of `golang:1.11beta2` ([docker image](https://hub.docker.com/_/golang/)) and are expected to become available in the GA version of `Go1.11` around August 1st (a day after this posting).
+Despite the growing acceptance of `dep` as _the_ dependency management tool for Golang, in mid-2018 a [new proposal](https://blog.golang.org/versioning-proposal) emerged for managing Golang project dependencies, known as `vgo` or Versioned Go Modules. The proposal was accepted, and as of [`Go1.11`](https://tip.golang.org/doc/go1.11#modules) modules will be available as an alternative to `GOPATH`, with integrated support for versioning and package distribution. This support is currently available as part of `golang:1.11.0` ([docker image](https://hub.docker.com/_/golang/)) and all other v1.11 GA releases.
```

Comment:
I would go further and and just say "as of Go1.11 modules are available as an alternative to GoPATH... package distribution." [end paragraph]